server/pbench/bin/pbench-inotify

```diff
@@ -219,6 +224,10 @@ def main():
     # initialization period: any pbench-move-results will have to wait
     # for this to finish.
 
+    wd1=wm.add_watch("/pbench/archive/fs-version-001", mask)
```

Comment:
This has to agree with the path that we give to `pbench-move-results` (iow, what we put in the `pbench-results-host-info` file). We should specify this path in the config file and retrieve it from there, both in this script and also in the iinstallation script that creates that file.